test/viewmodels/IndexPageHelperSpec.scala

```diff
@@ -1196,7 +1196,8 @@ class IndexPageHelperSpec extends SpecBase {
                 LPPDetailsMetadata = LPPDetailsMetadata(
                   mainTransaction = Some(MainTransactionEnum.VATReturnCharge),
                   outstandingAmount = Some(99)
-                )
+                ),
+                timeToPay = None //TODO Create case for user with TTP (PRM-1721)
```

Comment:
Should this be implemented in PRM-1720 instead?

Reply:
Correct Updated ✅